retro/sadtalker.py

```diff
@@ -151,6 +211,24 @@ def sadtalker(
             print("\t$ " + " ".join(args))
             print(subprocess.check_output(args, encoding="utf-8"))
             audio_path = wav_audio_path
+        # make sure audio is not 0 seconds
+        audio_length = float(
```

Comment:
can you write it like we do ffprobe_video

Reply:
[b18f980](https://github.com/GooeyAI/gooey-gpu/pull/26/commits/b18f980576e1fe8429cf4dd9b58712d5fc9bda2d)